ui/client/components/table/TableSelect.tsx

```diff
@@ -2,7 +2,16 @@ import React from "react"
 import Select from "react-select"
 import "../../stylesheets/processes.styl"
 
-export default class TableSelect extends React.Component {
+type Props = {
+  defaultValue: $TodoType,
+  onChange: $TodoType,
+  options: $TodoType[],
+  isMulti: boolean,
+  isSearchable: boolean,
+  placeholder: string,
+}
+
+export default class TableSelect extends React.Component<Props> {
```

Comment:
`React.Component` need to params: Props and State. IDEA shows me error with missing State..

Reply:
no, `interface Component<P = {}, S = {}, SS = any> extends ComponentLifecycle<P, S, SS> { }`,  check your config.

btw changed to FC later